src/adaptors/adaptor_common.c

```diff
@@ -66,7 +66,9 @@ qd_error_t qd_load_adaptor_config(qdr_core_t *core, qd_adaptor_config_t *config,
     config->site_id = qd_entity_opt_string(entity, "siteId", 0);               CHECK();
     config->ssl_profile_name  = qd_entity_opt_string(entity, "sslProfile", 0); CHECK();
     config->authenticate_peer = qd_entity_opt_bool(entity, "authenticatePeer", false); CHECK();
-    config->verify_host_name  = qd_entity_opt_bool(entity, "verifyHostname", false);   CHECK();
+    config->verify_host_name  = qd_entity_opt_bool(entity, "verifyHostname", false);
+    CHECK();
+    CHECK();
```

Comment:
There are two calls to `CHECK();`, should only be one. This file `src/adaptors/adaptor_common.c` should not appear in the list of changed files because the the change to obtain `terminate_tcp_conns` flag is in `dispatch.c`

Reply:
I removed the extra CHECK(). The file still is in the changeset because `git clang-format` moves the `CHECK()` to the next line.